package.json

```diff
@@ -37,7 +37,7 @@
     "test": "npm run lint && npm run check && npm run testonly:cover && npm run prettier:check && npm run check:spelling && npm run check:integrations",
     "lint": "eslint --cache --max-warnings 0 --rulesdir resources/eslint-internal-rules/ .",
     "check": "tsc --pretty",
-    "testonly": "mocha --full-trace src/**/__tests__/**/*-test.ts",
+    "testonly": "dotenvx run --quiet --env NODE_ENV=development -- mocha --full-trace src/**/__tests__/**/*-test.ts",
```

Comment:
Does mocha set NODE_ENV=test by default? If so, this shouldn't be needed.

Reply:
it does not... tested locally without dotenvx and fails

also, see:
https://github.com/mochajs/mocha/issues/185

thinking from their end is that you might want to use mocha to test in multiple environments, best if mocha stays out of your env